### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by kingqabil**
Added Index file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                <a href="https://twitter.com/kingqabil">
+                    <i class="fab fa-twitter"></i>
+                </a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/sadiq-habil-802892130/">
+                    <i class="fab fa-linkedin"></i>
+                </a>
+            </li>
+        </ul>
```
Comment (elisha2kyakpo1) [inline index.html:65]: - [ ] The social icons are not properly aligned, and also look different from the Figma design.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        </div>
+    </header>
+
+    <!--End Navbar-->
+    <!-- Headline -->
+
+    <section class="headline">
+        <h1 class="crete-font">Hey There. I'm SadiQ</h1>
+        <h2 class="inter-font">I'am a Software Developer</h2>
+        <p class="inter-font">
```
Comment (elisha2kyakpo1) [inline index.html:38]: - [ ] I couldn't find the design you used, I advise you to follow the design given as per project instructions. I guess you picked the design below, if so then you should follow all the design specifications including colors, typography, etc.

Figma

Your project

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi, @kingqabil,

You have done a great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there


## Required Changes ♻️

- [ ] Edit the PR summary to include other tasks you did in this branch. 

summary

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- The PR title should be more descriptive, for example, Portfolio: setup and mobile version skeleton

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,42 @@
-# My-Portofolio
\ No newline at end of file
+
+# My-Portfolio
+
+> My Second Project My Portfolio Website.
+
+![screenshot](./desktop.png)
+![screenshot](./mobile.png)
```
Comment (jmontanholi) [inline README.md:7]: # [OPTIONAL]

- [ ] Please consider changing these images to new ones that correspond to your project's current state. 👍🏻

**Review by jmontanholi — CHANGES_REQUESTED** [review]
Hello @kingqabil! 

You have done a great job so far, but there are still some things that you can improve before going to the next project. 😄 

✅ You fixed the design colors and schema
☑️ You fixed the icons

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please, as the other reviewer said, consider changing your Pull Request description to include what work you did in this branch. 👍🏻 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, working on optional suggestions is always good, with them you may learn something new or make your project look more professional._

You can also consider:
- Please consider changing the Pull Request title to a more descriptive one, this way by looking at its title people will be able to know what work is being done on it. 👍🏻 
- Look at the optional inline comment, please.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,68 @@
...
+## Built With
+
+- HTML & CSS
+- GitHub Workflows
+-Figma
+-Browser Developer Tools.
+
+## Live Demo
+
+[Live Demo Link]https://github.com/kingqabil/My-Portfolio/feature2/index.html)
```
Comment (BrianSammit) [inline README.md:18]: - [ ] [OPTIONAL] Please fix this link. You just forget to open the braces. Your readme look like this `[Live Demo Link]https://github.com/kingqabil/My-Portfolio/feature2/index.html)` and it should look like this  [Live Demo Link](https://github.com/kingqabil/My-Portfolio/feature2/index.html)

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @kingqabil, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 


## Optional suggestions 👌 

- [ ] [OPTIONAL] It would be so much better to write a more descriptive summary. The pull request summary is meant to help your team ( In this case the reviewers) to know what features you added and want to merge into the main branch.  Instead of just write `Added Index file` write something like 
```
In this portfolio project I added: 
- The toolbar
- The headline section
Using template 2 from Figma
```
I highly recommend you be more descriptive in your pull request description. This make your repo more [professional](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

Check my inline comments 👇 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**